### DISCUSSION TITLE
Add custom button with modal and action near Search Bar

### DISCUSSION THREAD

**Discussion by HomaEEE** [filamentphp/filament#3334] [Help] [answered]
I need add button to header, for show modal and add data to DB in subsequently

<img width="1430" alt="Снимок экрана 2022-08-01 в 16 24 16" src="https://user-images.githubusercontent.com/1429343/182158038-183428f2-d262-4d3c-b098-311615f53401.png">

**Comment by danharrin** [comment, marked as answer]
`global-search.start` render hook? https://filamentphp.com/docs/2.x/admin/appearance#render-hooks

**Comment by HomaEEE** [comment]
I know about render hook, but how add mods with form? Via Livewire?

**Reply by danharrin** [reply]
Yeah, render a Livewire component with an Alpine.js modal, with the Filament form builder inside

**Reply by HomaEEE** [reply]
may be have example of code or add same function to demo repo?

**Reply by danharrin** [reply]
There are lots of plugin examples

https://github.com/awcodes/filament-quick-create
https://github.com/bezhanSalleh/filament-language-switch
https://github.com/pxlrbt/filament-environment-indicator
